### ISSUE TITLE
SQL Alchemy is not properly executing valid SQL that contains a % character

### ISSUE THREAD

**Issue by thethingstheycoded** [dagster-io/dagster#184]
`dict' object does not support indexing` error generated by SQL Alchemy

Putting the following text into a SQL Solid generates an error in sql alchemy, due to the `%` character:
```
SELECT * FROM table_name WHERE column_name LIKE '%partial_value%'
```

The error message generated is below.  Note that this error was generated within a private project, so the message has been slightly edited to enable sharing publicly.
```
2018-09-14 20:56:50 - dagster - ERROR - message="'dict' object does not support indexing" data_source_run_id=4f1aa827-712d-4c7e-bdee-7d60edbe6a14 data_source=data_source pipeline_run_id=86668094-a62d-4f24-87f4-4024d9948458 pipeline=pipeline_name solid=solid_name sql_text="
SELECT * FROM table_name WHERE column_name LIKE '%partial_value%'" stack_trace="  File "anaconda3/lib/python3.6/site-packages/dagster/core/execution.py", line 316, in _user_code_error_boundary
    yield
  File "anaconda3/lib/python3.6/site-packages/dagster/core/execution.py", line 417, in _execute_core_transform
    transformed_value = solid_transform_fn(context, values_dict)
  File "project/common.py", line 73, in do_transform
    on_error(context, e)
  File "project/pipeline.py", line 61, in _error_func
    raise e
  File "data_source/common.py", line 70, in do_transform
    execute_sql_text_on_context(context, rendered_sql)
  File "anaconda3/lib/python3.6/site-packages/dagster/sqlalchemy_kernel/common.py", line 79, in execute_sql_text_on_context
    connection.execute(sql_text)
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 942, in execute
    return self._execute_text(object, multiparams, params)
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1104, in _execute_text
    statement, parameters
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1416, in _handle_dbapi_exception
    util.reraise(*exc_info)
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 187, in reraise
    raise value
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "anaconda3/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 508, in do_execute
    cursor.execute(statement, parameters)
"
```

The is to pass a SqlAlchemy `TextClause` into the SqlAlchemy `execute` function instead of raw text, as indicated here: https://stackoverflow.com/questions/8657508/strange-sqlalchemy-error-message-typeerror-dict-object-does-not-support-inde

Sql Alchemy TextClause documentation is here: http://docs.sqlalchemy.org/en/latest/core/sqlelement.html?highlight=sqlalchemy%20text#sqlalchemy.sql.expression.text

**Comment by mgasner** [comment]
is this resolved in #182 -- can we close this issue? @schrockn @thethingstheycoded

**Comment by mgasner** [comment]
Closing per comment on #182, feel free to reopen if I'm mistaken.
